Client/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -30,15 +47,16 @@ class SyncSettingsTableViewController: UITableViewController {
         super.viewDidLoad()
         title = Strings.sync
         
-        frc = Device.frc()
-        frc?.delegate = self
-        
-        do {
-            try frc?.performFetch()
-        } catch {
-            log.error("frc fetch error: \(error)")
+        syncDeviceObserver = BraveSyncDeviceObserver { [weak self] in
+            self?.updateDeviceList()
         }
         
+        let codeWords = BraveSyncAPI.shared.getSyncCode()
+        BraveSyncAPI.shared.joinSyncGroup(codeWords: codeWords)
+        BraveSyncAPI.shared.syncEnabled = true
```

Comment:
why do we join and turn on sync here?

Reply:
Because that's how the API works. It's the same API for both creating and joining a sync chain.
The only way to start the sync engine, is to setSyncEnabled.

You can never get to this screen unless you were already in a sync chain as per the `Preferences` anyway.

Previous SyncV1 code was using Keychain to store the words and it'd retrieve the words there to determine if the user was already in a sync chain.

In SyncV2, we use preferences and we use the BraveCoreAPI for getting the code words.